prepare_llm_models.sh

```diff
@@ -30,26 +30,22 @@ if [ "$2" = "docker" ]; then
     python3 -m pip install $(find wheel -name 'openvino_tokenizers*.whl')
     python3 -m pip install "optimum-intel"@git+https://github.com/huggingface/optimum-intel.git nncf sentence_transformers==3.1.1
 else
-    python3 -m venv .venv
+    python3.10 -m venv .venv
     . .venv/bin/activate
     pip3 install -U pip
-    pip3 install --pre -U "optimum-intel[nncf,openvino]"@git+https://github.com/huggingface/optimum-intel.git openvino-tokenizers sentence_transformers==3.1.1
+    pip3 install -U -r demos/common/export_models/requirements.txt
 fi
+mkdir -p $1
 
 if [ -d "$1/facebook/opt-125m" ]; then
   echo "Models directory $1/facebook/opt-125m exists. Skipping downloading models."
 else
-  optimum-cli export openvino --disable-convert-tokenizer --model facebook/opt-125m --weight-format int8 $1/facebook/opt-125m
-  convert_tokenizer -o $1/facebook/opt-125m --with-detokenizer --skip-special-tokens --streaming-detokenizer facebook/opt-125m
+  python demos/common/export_models/export_model.py text_generation --source_model facebook/opt-125m --weight-format int8 --model_repository_path $1
```

Comment:
it would be nice to use constant for "facebook/opt-125m", like for EMBEDDING_MODEL